.github/workflows/deploy-to-store-and-wporg.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - "*"
+jobs:
+  tag:
+    name: New version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get The Version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+      - name: Run Composer Install
+        run: |
+          npm ci
+          composer install --no-dev --prefer-dist --no-progress --no-suggest
+      - name: WordPress Plugin Deploy
+        uses: 10up/action-wordpress-plugin-deploy@master
+        env:
+          SVN_PASSWORD: ${{ secrets.SVN_THEMEISLE_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_THEMEISLE_USERNAME }}
+      - name: Send Update to the Store
```

Comment:
we don't need this step since there is no download on the rs store for this product, only for the pro version.

Reply:
> we don't need this step since there is no download on the rs store for this product, only for the pro version.

Made the change here: https://github.com/Codeinwp/tweet-old-post/pull/810/commits/261260894db9b4a5294f8e948baf4fb141e3fdea